src/code/z_actor_dlftbls.c

```diff
@@ -37,6 +39,24 @@
 #define DEFINE_ACTOR_INTERNAL(name, _1, allocType, nameString) \
     { 0, 0, NULL, NULL, NULL, &name##_InitVars, nameString, allocType, 0 },
 
+#else
+
+#define DEFINE_ACTOR(name, _1, allocType, nameString) \
+    { (uintptr_t)_ovl_##name##SegmentRomStart,        \
```

Comment:
to follow the convention i suppose nameString should become _3 because its now an unused arg